.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@ that is provided, the better.
 
 **Description of the changes being introduced by the pull request**:
 
-**Please verify and check that the pull request fulfills the following
+**Please verify and check that the pull request fulfils the following
```

Comment:
I dunno, this seems to be a British vs American English thing... 😅

Reply:
Oh no! 😬